code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -352,23 +353,17 @@
 				redshade_A.speak = speak
 				redshade_B.speak = speak
 				target = deconvertee
+				spawn(1)
+					redshade_A.forceMove(get_turf(pick(adjacent_turfs)))
+					redshade_B.forceMove(get_turf(pick(adjacent_turfs)))
+					redshade_A.GiveTarget(target)
+					redshade_B.GiveTarget(target)
+					redshade_A.MoveToTarget()
+					redshade_B.MoveToTarget()
 			if (DECONVERSION_REFUSE)
 				playsound(deconvertee, 'sound/effects/deconversion_failed.ogg', 50, 0, -4)
 				to_chat(deconvertee,"<span class='notice'>You manage to block out the exorcism.</span>")
-				deconvertee.visible_message("<span class='userdanger'>The ritual was resisted, a pair of shades manifest and start attacking all nearby.</span>","<span class='warning'>The energies you mustered take their toll on your body, and manifest into a couple or red shades that start attacking whoever tried to deconvert you.</span>")
-				var/list/speak = list("...how dare you try and harass [deconvertee]...","...this is a blatant disregard of the freedom of religion...","...[deconvertee] has pledged their blood to Nar-Sie and we demand that you respect their choice...")
-				if (cult_chaplain)
-					speak = list("...cut it out with the weird fanfictions [deconverter]...","...that is why we don't want you among us...","...go back to do word research where no one can hear about you [deconverter]...")
-				redshade_A.speak = speak
-				redshade_B.speak = speak
-				target = deconverter
-		spawn(1)
-			redshade_A.forceMove(get_turf(pick(adjacent_turfs)))
-			redshade_B.forceMove(get_turf(pick(adjacent_turfs)))
-			redshade_A.GiveTarget(target)
-			redshade_B.GiveTarget(target)
-			redshade_A.MoveToTarget()
-			redshade_B.MoveToTarget()
+				deconvertee.visible_message("<span class='userdanger'>The ritual was resisted!</span>","<span class='warning'>The energies you mustered take their toll on your body...</span>")
```

Comment:
what toll?

Reply:
"deconvertee.take_overall_damage(10)//it's a painful process no matter what."

Reply:
alzheimer